controllers/machinehealthcheck_controller.go

```diff
@@ -288,7 +288,7 @@ func (r *MachineHealthCheckReconciler) reconcile(ctx context.Context, logger log
 	conditions.MarkTrue(m, clusterv1.RemediationAllowedCondition)
 
 	errList := r.PatchUnhealthyTargets(ctx, logger, unhealthy, cluster, m)
-	errList = append(errList, r.PatchHealthyTargets(ctx, logger, healthy, cluster, m)...)
+	errList = append(errList, r.PatchHealthyTargets(ctx, logger, healthy, m)...)
```

Comment:
Why was the `cluster` removed from this list in this change? Is the cluster not actually used in the `PatchHealthyTargets`?